between random: 3550007
between random: 3550002
between random: 3550007
between random: 11282
"    require conn_seen(dcid) -> hi_non_probing_endpoint(dcid,dst);  # [10]
"
/usr/local/lib/python2.7/dist-packages/ivy/include/1.7/quic_packet.ivy: line 397: error: assumption failed
output directory: temp/11717
implementation directory: /quic/aioquic
implementation command: python3 examples/http3_server.py --certificate /quic/aioquic/tests/ssl_cert.pem --private-key /quic/aioquic/tests/ssl_key.pem  -v --host 127.0.0.1 --port 4443 -l /results/temp/aioquic_key.log"
../quic_server_test_accept_maxdata (0) ...
implementation command: ['python3', 'examples/http3_server.py', '--certificate', '/quic/aioquic/tests/ssl_cert.pem', '--private-key', '/quic/aioquic/tests/ssl_key.pem', '-v', '--host', '127.0.0.1', '--port', '4443', '-l', '/results/temp/aioquic_key.log"']
quic_process pid: 28111
timeout 100 ./build/quic_server_test_accept_maxdata seed=605 the_cid=0 server_cid=1 client_port=4987 client_port_alt=4988
client return code: 1
FAIL
error: 1 tests(s) failed
